### PR TITLE
Bugfix #11430 [v105]: fix minor spellings in swift comments

### PR DESCRIPTION
**Pull request by byaruhaf**
The are a few minor misspelled words in the swift comments of the following swift files

Client/Application/AppDelegate.swift
Client/Application/AppLaunchUtil.swift
Client/Experiments/Messaging/GleanPlumbMessageStore.swift
Client/Experiments/UserResearch.swift
Client/FeatureFlags/FeatureFlagsManager.swift
Client/Frontend/Browser/BrowserViewController.swift
Client/Frontend/Browser/DefaultSearchPrefs.swift
Client/Frontend/Browser/GridTabViewController.swift
Client/Frontend/Browser/SearchLoader.swift
Client/Frontend/Browser/TabManager.swift
Client/Frontend/Browser/Tabs/RemoteTabsPanel.swift
Client/Frontend/Browser/TopTabsViewController.swift
Client/Frontend/Components/ActionButton.swift
Client/Frontend/Home/HistoryHighlights/HistoryHighlightsManager.swift
Client/Frontend/Home/HistoryHighlights/HistoryHightlightsViewModel.swift
Client/Frontend/Home/HomepageTabBanner.swift
Client/Frontend/Home/HomepageViewModel.swift
Client/Frontend/Home/Wallpapers/Legacy/Utilities/LegacyWallpaperStorageUtility.swift
Client/Frontend/InternalSchemeHandler/ErrorPageHelper.swift
Client/Frontend/Intro/IntroViewController.swift
Client/Frontend/Reader/ReadabilityService.swift
Client/Frontend/Settings/AppSettingsOptions.swift
Client/Frontend/Settings/LoginDetailViewController.swift
Client/Frontend/Settings/SearchSettingsTableViewController.swift
Client/Frontend/UXSizeClass.swift
Client/Frontend/Widgets/AutocompleteTextField.swift
Client/Helpers/TabEventHandler.swift
Client/Protocols/ReusableCell.swift
Client/Protocols/URLCaching.swift
Client/Telemetry/TelemetryWrapper.swift
Client/Utils/ConfigurableGradientView.swift
Client/Utils/FaviconFetcher.swift
Client/Utils/Layout.swift
Client/Utils/SearchTermGroupsUtility.swift
Providers/TopSitesProvider.swift
RustFxA/RustFirefoxAccounts.swift
Shared/DeferredUtils.swift
Storage/History.swift
Storage/SQL/BrowserSchema.swift
Storage/SQL/SQLiteHistory.swift
Storage/Visit.swift
Sync/Synchronizers/HistorySynchronizer.swift
Tests/ClientTests/HistoryHighlights/HistoryHighlightsManagerTests.swift
Tests/ClientTests/TabManagerTests.swift
Tests/SharedTests/UtilsTests.swift
Tests/StorageTests/TestSQLiteHistory.swift
Tests/StorageTests/TestSQLiteRemoteClientsAndTabs.swift
Tests/UITests/LoginManagerTests.swift
Tests/UITests/ReopenLastTabTests.swift
Tests/XCUITests/ActivityStreamTest.swift
Tests/XCUITests/BookmarkingTests.swift
Tests/XCUITests/DesktopModeTests.swift
Tests/XCUITests/DisplaySettingsTests.swift
Tests/XCUITests/DownloadFilesTests.swift
Tests/XCUITests/DragAndDropTests.swift
Tests/XCUITests/FindInPageTest.swift
Tests/XCUITests/FxScreenGraph.swift
Tests/XCUITests/HistoryTests.swift
Tests/XCUITests/HomePageSettingsUITest.swift
Tests/XCUITests/NavigationTest.swift
Tests/XCUITests/SaveLoginsTests.swift
Tests/XCUITests/ScreenGraphTest.swift
Tests/XCUITests/TabsPerformanceTests.swift
Tests/XCUITests/ToolbarTest.swift

### CODE REVIEW

**Comment by lmarceau** [comment]
Linking issue #11430 and JIRA https://mozilla-hub.atlassian.net/browse/FXIOS-4643

**Comment by thatswinnie** [comment]
@byaruhaf Wow, so many spelling mistakes. Thank you for catching and fixing them! 🙏 

I saw that you changed `frecency` to `frequency`. Actually frecency is the correct word. Here is the meaning of it:

> In [computing](https://en.wikipedia.org/wiki/Computing), frecency is any [heuristic](https://en.wikipedia.org/wiki/Heuristic) that combines the frequency and [recency](https://en.wikipedia.org/wiki/Recency) into a single measure. 
[Wikipedia](https://en.wikipedia.org/wiki/Frecency

Would you mind changing these back?

**Comment by byaruhaf** [comment]
let me make the changes didn't know about frecency, learned something new.

**Comment by byaruhaf** [comment]
Changes done.

File: Storage/History.swift [hunk 1]
```diff
@@ -40,7 +40,7 @@ public protocol BrowserHistory {
 }
 
 /**
- * An interface for fast repeated frecency queries.
+ * An interface for fast repeated frequency queries.
```
Comment (thatswinnie) [inline Storage/History.swift:43]: Could you change this one back as well, please?

File: Storage/SQL/SQLiteHistory.swift [hunk 2]
```diff
@@ -163,7 +163,7 @@ private let topSitesQuery = """
         """
 
 /**
- * The init for this will perform the heaviest part of the frecency query
+ * The init for this will perform the heaviest part of the frequency query
```
Comment (thatswinnie) [inline Storage/SQL/SQLiteHistory.swift:166]: Could you change this one back as well, please?

File: Storage/SQL/SQLiteHistory.swift [hunk 3]
```diff
@@ -301,9 +301,9 @@ private struct SQLiteFrecentHistory: FrecentHistory {
             LIMIT 1000
             """
 
-        // Next: merge by domain and select the URL with the max frecency of a domain, ordering by that sum frecency and reducing to a (typically much lower) limit.
-        // NOTE: When using GROUP BY we need to be explicit about which URL to use when grouping. By using "max(frecency)" the result row
-        //       for that domain will contain the projected URL corresponding to the history item with the max frecency, https://sqlite.org/lang_select.html#resultset
+        // Next: merge by domain and select the URL with the max frequency of a domain, ordering by that sum frecency and reducing to a (typically much lower) limit.
+        // NOTE: When using GROUP BY we need to be explicit about which URL to use when grouping. By using "max(frequency)" the result row
+        //       for that domain will contain the projected URL corresponding to the history item with the max frequency, https://sqlite.org/lang_select.html#resultset
```
Comment (thatswinnie) [inline Storage/SQL/SQLiteHistory.swift:306]: Could you change all of these one back as well, please?

File: Storage/SQL/SQLiteHistory.swift [hunk 4]
```diff
@@ -285,8 +285,8 @@ private struct SQLiteFrecentHistory: FrecentHistory {
 
         // Next: limit to only those that have been visited at all within the last six months.
         // (Don't do that in the innermost: we want to get the full count, even if some visits are older.)
-        // Discard all but the 1000 most frecent.
-        // Compute and return the frecency for all 1000 URLs.
+        // Discard all but the 1000 most frequent.
+        // Compute and return the frequency for all 1000 URLs.
```
Comment (thatswinnie) [inline Storage/SQL/SQLiteHistory.swift:289]: Could you change these one back as well, please?

**Review by thatswinnie — COMMENTED** [review]
Awesome! Thanks @byaruhaf . Unfortunately you missed some. Could you address these as well, please?

**Comment by byaruhaf** [comment]
oops 🤦🏾‍♂️ not sure how I missed those,  but additional changes have been implemented.

**Review by thatswinnie — APPROVED** [review]
Thanks so much @byaruhaf - great work!

**Pull request merged**